src/components/formik-ui/Input/Textarea.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ChangeEvent, FocusEvent, forwardRef } from 'react';
+import { useField } from 'formik';
+
+import OriginalTextarea, {
+  TextareaProps as OriginalTextareaProps,
+} from '@/components/ui/Input/Textarea';
+
+type TextareaProps = FormikFieldBaseProps & OriginalTextareaProps;
+
+export default forwardRef<HTMLTextAreaElement, TextareaProps>(function Input(
+  { name, onChange, onBlur, ...rest }: TextareaProps,
+  ref
+): JSX.Element {
+  const [field, meta] = useField(name);
+  const { onChange: formikOnChange, onBlur: formikOnBlur } = field;
+  const { error, touched } = meta;
+  const finalOnChange = (e: ChangeEvent<HTMLTextAreaElement>) => {
```

Comment:
❗️
`useCallback` + давай отбивать "обычные" переменные и функции пустой строкой.
Пройдись по всем файлам, где вниз спускаешь коллбэки и оберни их в `useCallback`
```suggestion
  const { error, touched } = meta;
  
  const finalOnChange = useCallback((e: ChangeEvent<HTMLTextAreaElement>) => { ...
```

Reply:
Надеюсь на эти пустые строки тоже можно настроить eslint